apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
I would decrease initial Delay and increase timeout - bcs initial delay is static waiting, but timeout is periodically checked, thus it will provide better (faster) results.